src/_premake_init.lua

```diff
@@ -274,7 +274,7 @@
 		kind = "string",
 		allowed = {
 			"Default",
-			"c7",
+			p.C7,
```

Comment:
Is there a reason to have `p.C7` over treating it like most other API values? Since nothing else seems to work like that, I assume it's a relic of premake4 or even earlier and can probably just be removed instead (the variable `p.C7`, not the value `"c7"`).